src/DataProtection/README.md

```diff
@@ -170,8 +170,8 @@ directive:
   - where:
       verb: Get
       subject: BackupVault
-      variant: ^GetViaIdentity2$|^Get$|^GetViaIdentity1$
-    remove: true
+      variant: ^GetViaIdentity2$|^Get$|^GetViaIdentity1$|^Get1$
```

Comment:
please hide parameter `VaultName` and `InputObject` for `Get-AzDataProtectionBackupVault` and delete the customized cmdlet

Reply:
This custom command was already there and was removed in PR - https://github.com/Azure/azure-powershell/pull/15774 which caused the regression. We're fixing in this PR